dbscripts/xml/upgrade.xml

```diff
@@ -160,6 +160,7 @@
 		<migration class="APP\migration\upgrade\v3_5_0\I10620_EditorialBoardMemberRole"/>
 		<migration class="APP\migration\upgrade\v3_5_0\I9707_WeblateUILocales"/>
 		<note file="docs/release-notes/README-3.5.0" />
+		<migration class="PKP\migration\upgrade\v3_5_0\I10759_AddReviewAssignmentSettings"/>
```

Comment:
Sorry I didn't catch this before. This should go before the `<note>` tag, as that should be the last thing to run as part of the upgrade.